src/main/python/cfn_sphere/aws/cfn.py

```diff
@@ -251,43 +254,64 @@ def _create_stack(self, stack):
         Create cloudformation stack
         :param stack: cfn_sphere.aws.cfn.CloudFormationStack
         """
-        self.client.create_stack(
-            StackName=stack.name,
-            TemplateBody=stack.template.get_template_json(),
-            Parameters=stack.get_parameters_list(),
-            TimeoutInMinutes=stack.timeout,
-            Capabilities=[
+        kwargs = {
+            "StackName": stack.name,
+            "TemplateBody": stack.template.get_template_json(),
+            "Parameters": stack.get_parameters_list(),
+            "TimeoutInMinutes": stack.timeout,
```

Comment:
Has nothing to do with this particular change, but setting `TimeoutInMinutes` to the stack config `timout`, which uses seconds as unit confuses two different timeouts: The stack config `timeout` defines how long cfn-sphere should keep polling for events, where `TimeoutInMinutes` defines how long CloudFormation itself waits for the stack operation to finish.

`"TimeoutInMinutes": stack.timeout / 60` would be correct, but an incompatible change for many. I would just not set it, as the default timeout is not defined but depends on the type of resources used: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/troubleshooting.html